mobx_codegen/example/test/generator_example_test.dart

```diff
@@ -1,5 +1,5 @@
-import 'package:example/src/generator_example.dart';
 import 'package:mobx/mobx.dart';
+import '../lib/src/generator_example.dart';
```

Comment:
Should be `import 'package:mobx_codegen_example/src/generator_example.dart'`?

Reply:
It was failing with all these errors, if I did that:

```shell
 error • Target of URI doesn't exist: 'package:mobx_codegen_example/example.dart' at example/test/generator_example_test.dart:2:8 • uri_does_not_exist
  error • Undefined class 'User' at example/test/generator_example_test.dart:6:37 • undefined_class
  error • The function 'User' isn't defined at example/test/generator_example_test.dart:62:19 • undefined_function
  error • The function 'Admin' isn't defined at example/test/generator_example_test.dart:62:43 • undefined_function
  error • The function 'Admin' isn't defined at example/test/generator_example_test.dart:65:19 • undefined_function
  error • The function 'Admin' isn't defined at example/test/generator_example_test.dart:77:19 • undefined_function

```

Reply:
alright, leave it like that then :S